src/patternfly/components/Wizard/examples/Wizard.md

```diff
@@ -459,6 +459,157 @@ import './Wizard.css'
 {{/wizard}}
 ```
 
+### Error on step
+```hbs isFullscreen
+{{#> wizard wizard--id="wizard-basic"}}
+  {{#> wizard-header}}
+    {{> wizard-close}}
+    {{> wizard-title wizard-title-text--value="Wizard title"}}
+    {{#> wizard-description}}
+      Here is where the description goes
+    {{/wizard-description}}
+  {{/wizard-header}}
+  {{#> wizard-toggle}}
+    {{#> wizard-toggle-list}}
+        {{#> wizard-toggle-list-item}}
+          {{#> wizard-toggle-num}}2{{/wizard-toggle-num}}
+          Configuration
```

Comment:
Think this list item needs the status icon passed in as well. On wider viewport the 2nd step has an error icon, but on smaller viewport the toggle doesn't show the error icon. That or the 2nd step for desktop view should have the error icon removed (so that only the 1st step has an error).

Wider viewport:

![image](https://github.com/patternfly/patternfly/assets/70952936/ba79573d-7521-40da-96f8-267315026b21)

Smaller viewport:

![image](https://github.com/patternfly/patternfly/assets/70952936/b68c7174-a6b6-4d9d-a5ce-d77beee527ab)